src/zoning/specs/los-angeles.csv

```diff
@@ -50,25 +50,25 @@ R1,1,0,45,5000,1,5000,50,,,,,2,,,,,0-20,20,0-50,10,15,,
 R1R3,1,0,45,5000,1,5000,50,,,,,2,,,,0.5,0-20,20,0-50,10,15,,https://planning.lacity.org/ordinances/docs/R1VariationZones/QA_10-6-16.pdf
 R1V2,1,0,45,5000,1,5000,50,,,,,2,,,,0.46,0-20,20,0-50,10,15,,https://planning.lacity.org/ordinances/docs/R1VariationZones/QA_10-6-16.pdf
 R1V3,1,0,45,5000,1,5000,50,,,,,2,,,,0.42,0-20,20,0-50,10,15,,https://planning.lacity.org/ordinances/docs/R1VariationZones/QA_10-6-16.pdf
-R2,0,1,45,2500,2,5000,50,,,,,2,,,,,0-20,20,0-50,10,15,,
-R3,0,1,45,6000,,3000,60,,,,,1-3,,,,,10,,5-10,10,15,,
-R3(PV),0,1,45,6000,,3000,60,,,,,1-3,,,,,10,,5-10,10,15,,
-R4,0,1,85,5000,,400,50,,,,,1-3,,,,,10-15,,3-50,10,15-20,,
-R4(PV),0,1,85,5000,,400,50,,,,,1-3,,,,,10-15,,3-50,10,15-20,,
-R5,0,1,85,200,,5000,50,,,,,1-3,,,,,10-15,,3-50,10,15-20,,
-RA,1,0,45,175000,,17500,70,,,,,2,,,,,0-25,20,10-70,10,10,,
-RD1.5,0,1,45,1500,,5000,50,,,,,1-3,,,,,15,,3-50,,15,,
-RD2,0,1,45,2000,,5000,50,,,,,1-3,,,,,15,,3-50,,15,,
-RD3,0,1,45,3000,,6000,60,,,,,1-3,,,,,15,,5-10,10,15,,
-RD4,0,1,45,400,,8000,60,,,,,1-3,,,,,15,,5-10,10,15,,
-RD5,0,1,45,5000,,10000,70,,,,,1-3,,,,,20,,10,,25,,
-RD6,0,1,45,6000,,12000,70,,,,,1-3,,,,,20,,10,,25,,
+R2,1,1,45,2500,2,5000,50,,,,,2,,,,,0-20,20,0-50,10,15,,
+R3,1,1,45,800,,5000,60,,,,,1-3,,,,,10,,5-10,10,15,,
+R3(PV),1,1,45,800,,5000,60,,,,,1-3,,,,,10,,5-10,10,15,,
+R4,1,1,85,400,,5000,50,,,,,1-3,,,,,10-15,,3-50,10,15-20,,
+R4(PV),1,1,85,400,,5000,50,,,,,1-3,,,,,10-15,,3-50,10,15-20,,
+R5,1,1,85,200,,5000,50,,,,,1-3,,,,,10-15,,3-50,10,15-20,,
+RA,1,0,45,17500,,17500,70,,,,,2,,,,,0-25,20,10-70,10,10,,
+RD1.5,1,1,45,1500,,5000,50,,,,,1-3,,,,,15,,3-50,,15,,
+RD2,1,1,45,2000,,5000,50,,,,,1-3,,,,,15,,3-50,,15,,
+RD3,1,1,45,3000,,6000,60,,,,,1-3,,,,,15,,5-10,10,15,,
+RD4,1,1,45,400,,8000,60,,,,,1-3,,,,,15,,5-10,10,15,,
+RD5,1,1,45,5000,,10000,70,,,,,1-3,,,,,20,,10,,25,,
+RD6,1,1,45,6000,,12000,70,,,,,1-3,,,,,20,,10,,25,,
 RE11,1,0,45,11000,1,11000,70,,,,,2,,,,,25-inf,20,10,,,25,
 RE15,1,0,45,15000,1,15000,80,,,,,2,,,,,25-inf,20,10,,,25,
 RE20,1,0,45,20000,1,20000,80,,,,,2,,,,,25-inf,20,10,,,25,
 RE40,1,0,45,40000,1,40000,80,,,,,2,,,,,25-inf,20,10,,,25,
 RE9,1,0,45,9000,1,9000,60,,,,,2,,,,,25-inf,20,10,,,25,
-RMP,0,1,45,20000,1,,80,,,,,2,,,,,0-25,25,10,,25,25,Mobile Home Park
+RMP,1,1,45,20000,1,20000,80,,,,,2,,,,,0-25,25,10,,25,25,Mobile Home Park

```

Comment:
Are we considering mobile home parks to be multifamily housing here?

Reply:
Yes... should that not be true? It's for the full park, right?